### PR TITLE
Adds presentation page for GitHub organization

### PR DESCRIPTION
**Pull request by jlengrand**
## Summary

Adds nice presentation page for GitHub organization. See a live example here : https://github.com/adyen-examples/

Happy to receive feedback if needed :)

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+## 👋 Welcome to the [Adyen](https://www.adyen.com/) GitHub organization 👋
+
+<!-- ![The Adyen Logo](https://github.com/adyen-examples/.github/raw/main/images/logo.png) -->
+
+<img src="https://github.com/adyen-examples/.github/raw/main/images/logo.png" height="100" alt="The Adyen logo">
+
+This page contains the source code for all of our libraries and plug-ins as well as our Open API specifications. To find a list of example integrations for those, you can visit **[the Adyen examples GitHub page](https://github.com/adyen-examples)**.
```
Comment (rikterbeek) [inline profile/README.md]: could you add SDKs here as well

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+* 📺 [Checkout our Tech playlist on Youtube](https://www.youtube.com/watch?v=VPpTgsJbIhc&list=PL6agz7H5yEoaS-bF2gIwRwe_ApzqmW_QX)
+
+**🌈 Contributions are most welcome so feel free to reach out, post issues and / or propose improvements🦄**
+
+## Currently available resources
+
+### Libraries and SDKs
+
+* [Adyen Web Drop-in and Components](https://github.com/Adyen/adyen-web)
+
```
Comment (rikterbeek) [inline profile/README.md]: remove empty line

File: profile/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+* [Adyen API Library for PHP](https://github.com/Adyen/adyen-php-api-library)
+* [Adyen API Library for Java](https://github.com/Adyen/adyen-java-api-library)
+* [Adyen API Library for Node.js](https://github.com/Adyen/adyen-node-api-library)
+* [Adyen API Library for .NET](https://github.com/Adyen/adyen-dotnet-api-library)
+* [Adyen API Library for Go](https://github.com/Adyen/adyen-go-api-library)
+* [Adyen API Library for Python](https://github.com/Adyen/adyen-python-api-library)
+* [Adyen API Library for Ruby](https://github.com/Adyen/adyen-ruby-api-library)
+
```
Comment (rikterbeek) [inline profile/README.md]: remove empty line

File: profile/README.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+* 📰 [Our developer newsletter](https://www.adyen.com/newsletter/developers)
+* 📺 [Checkout our Tech playlist on Youtube](https://www.youtube.com/watch?v=VPpTgsJbIhc&list=PL6agz7H5yEoaS-bF2gIwRwe_ApzqmW_QX)
+
+**🌈 Contributions are most welcome so feel free to reach out, post issues and / or propose improvements🦄**
+
+## Currently available resources
+
+### Libraries and SDKs
+
+* [Adyen Web Drop-in and Components](https://github.com/Adyen/adyen-web)
```
Comment (rikterbeek) [inline profile/README.md:29]: somehow the markup of this list is different then the one below

Comment (jlengrand) [reply profile/README.md:29]: Hum, I don't see what you mean. Can you propose a change maybe?

**Comment by jlengrand** [comment]
@rikterbeek Ping, could we merge please?

**Pull request merged**